astro/src/content/blog/authn-authz-deployment-models-cerbos-webinar.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2025-04-24
+title: How To Choose The Right AuthZ And AuthN Deployment Model
```

Comment:
Maybe this is already a set-in-stone title, but if not, for SEO, fully writing out Authorization and Authentication in the title would result in better discovery.